test/extensions/filters/http/fault/fault_filter_integration_test.cc

```diff
@@ -3,6 +3,8 @@
 
 #include "gtest/gtest.h"
 
+using Envoy::Http::HeaderValueOf;
```

Comment:
?

Reply:
This `using` is used to ensure tests of fault filter can be passed. 
I haven't found the reason why the original fault filter test passed. It originally neither used `using` nor `Http::HeaderValueOf`, but directly used `HeaderValueOf`. In my opinion, because the namespace is completely different, it should not be able to use this name directly.

Reply:
I know but you didn't change the test content, so why you needed to add this here?

Reply:
Because when I reworked the `LowerCaseString`, the test of the fault filter made an error during the compilation, and it prompted me to change the `HeaderValueOf` to `Http::HeaderValueOf`.

This prompt is correct, but I can't explain why the compilation can pass before modifying `LowerCaseString`.

Reply:
@lizan I get it.

Before modifying `Http::LowerCaseString`, `Http::LowerCaseString` is normal class in `Http` namespace. The `HeaderValueOf` call in fault filter test can be handled correctly because ADL. Check https://en.cppreference.com/w/cpp/language/adl.

After modifying `Http::LowerCaseString`, `Http::LowerCaseString` become a specialization of `Envoy::ValidatedLowerCaseStr` template which changed the lookup scope of `HeaderValueOf`. So we must use `using` declaration or udpate `HeaderValueOf` to `Http::HeaderValueOf`